spec/features/admin/customers_spec.rb

```diff
@@ -100,33 +108,13 @@
           allow(OpenFoodNetwork::FeatureToggle)
             .to receive(:enabled?).with(:customer_balance, user) { true }
 
-          create(
-            :order,
-            total: 0,
-            payment_total: 88,
-            distributor: managed_distributor1,
-            user: nil,
-            state: 'complete',
-            customer: customer1
-          )
-          create(
-            :order,
-            total: 99,
-            payment_total: 0,
-            distributor: managed_distributor1,
-            user: nil,
-            state: 'complete',
-            customer: customer2
-          )
-          create(
-            :order,
-            total: 0,
-            payment_total: 0,
-            distributor: managed_distributor1,
-            user: nil,
-            state: 'complete',
-            customer: customer4
-          )
+          # calling the variables necessary for this testcase here;
+          # 'paymen_method' and 'payment1' are necessary to add additional payments to an order, as in a test case below (see payment2)
+          payment_method
+          payment1
+          order1
+          order2
+          order3
```

Comment:
that's what `let!` is for. with it, these factories get executed in-order, instead of lazily as happens with `let`. Then, we won't need these calls.

Reply:
The reason why I used `let` was to comply with the previous state of this spec - before my first commit, these orders were created (not as variables and as such, not declared) inside this "describe" block. Tbh, I don't know why this was the case.

But yes, declaring these variables with `let!` makes sense :+1: